packages/@lwc/node-reactions/src/types.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+export type ReactionType = 1 /* connected */ | 2 /* disconnected */;
+
+export type ReactionCallback = (this: Element, reactionType: ReactionType) => void;
```

Comment:
I am changing this to pass the element as the first argument instead of the context.
The original motivation was to model it similar to addEventlistener(). For a listener, the listener is invoked with the element as the context.